include/swift/SIL/SILConstants.h

```diff
@@ -196,24 +195,30 @@ class SymbolicValue {
     return kind != Unknown && kind != UninitMemory;
   }
 
-  static SymbolicValue getUnknown(SILNode *node, UnknownReason reason) {
+  static SymbolicValue
+  getUnknown(SILNode *node, UnknownReason reason,
+             const llvm::SmallVector<SourceLoc, 4> &callStack) {
     assert(node && "node must be present");
     SymbolicValue result;
     result.representationKind = RK_Unknown;
     result.value.unknown = node;
+    result.aux.callStack = callStack;
     result.aux.unknown_reason = reason;
     return result;
   }
 
-  bool isUnknown() const {
-    return getKind() == Unknown;
-  }
+  bool isUnknown() const { return getKind() == Unknown; }
 
   /// Return information about an unknown result, including the SIL node that
   /// is a problem, and the reason it is an issue.
   std::pair<SILNode *, UnknownReason> getUnknownValue() const {
     assert(representationKind == RK_Unknown);
-    return { value.unknown, aux.unknown_reason };
+    return {value.unknown, aux.unknown_reason};
```

Comment:
Please split formatting changes out from behavior changing patches :-)

Reply:
Way ahead of you, my git-fu has rusted over the past year though :) Thanks for the comments, will address soon.